includes/migrations/class-delete-active-webhook.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Class Delete_Active_Webhook
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Migrations;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class Delete_Active_Webhook
+ *
+ * Fires an event on plugin upgrade to delete the avtive webhook.
+ * Runs only once. We want to know whether existing install had it
+ * enabled before the current version.
+ */
+class Delete_Active_Webhook {
+	/**
+	 * Checks whether we should trigger the event.
+	 */
+	public static function maybe_delete() {
+		if ( version_compare( get_option( 'woocommerce_woocommerce_payments_version' ), '4.8.0', '>' ) ) {
```

Comment:
Should this be `4.9.0` instead, since that's the next scheduled release?

Reply:
Good call on the next version!  I think we should update it to >= 4.9.0 because the logic here is to compare 4.9.0 with the version they are on.